src/components/DescriptionList/_description-list.scss

```diff
@@ -9,5 +9,9 @@
 
 .cc-description-list__definition {
   display: block;
-  margin: 0 0 var(--space-sm) 0;
+  margin: 0 0 var(--space-md) 0;
+}
+
+.cc-description-list__definition-inner {
+  display: inline;
 }
```

Comment:
I'm not 100% sure this is necessary. See screenshot (happy to be proven wrong!)

![Screenshot 2021-03-10 at 14 43 23](https://user-images.githubusercontent.com/8282383/110646810-2d0cb700-81af-11eb-8270-b2695b2b7096.png)

Reply:
Without this, the display breaks when longer blocks of text run into multiple lines.

Reply:
Good point.

Fwiw, I think using `float` is the wrong approach. It takes the element out of the flow and causes the need for more code to rectify (e.g. `clear: both`).

Would suggest using `display: inline` with a wrapping div. Will avoid layout issues I think.

```html
<dl>
  <div>
    <dt style="display: inline">term title</dt>
    <dd style="display: inline"><div class="cc-rich-text">term data</div></dd>
  </div>
 </dl>
```

![Screenshot 2021-03-10 at 15 49 06](https://user-images.githubusercontent.com/8282383/110656847-3ea68c80-81b8-11eb-9d5f-15a8576fe0b0.png)

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl#wrapping_name-value_groups_in_htmlelementdiv_elements

Reply:
Hey @chriddell - finally got round to looking at this.

Re. float, I see your point, but I'd argue the float doesn't need clearing as it's proceed by a block level level element that isn't floated. 

I've just tried to run with what you're suggesting and it's not providing the layout required.

If it's okay with you, I'd rather go with the current approach. While the HTML5 spec does allow a `div` to be added here, it feels that we're using deliberately adding markup to influence the styling.

Would you be okay with this?

Reply:
No worries - if you've tried the approach I suggested and still think yours is better, am totally happy for you to go ahead with that